### PR TITLE
Finish user creation form front end

### PR DESCRIPTION
**Pull request by danyliu9**
image

Should be easy to hook up to the back end as soon as the endpoint for user creation is added, for now I just changed the baseUrl of the the users service to work with our temporary json-server.

### CODE REVIEW

File: packages/client/src/pages/common/nav/create-profile-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      <DialogContent className={classes.dialogContent}>
+        <DialogContentText>
+          Create your Alpacow account. {<br />}
+          Keep in mind, your username represents how others see you on Alpacow.
+        </DialogContentText>
+        <TextField
+          id="new-user-email"
+          label="Email"
+          // TODO: Make this style reusable between these elements? (I don't know how I should do it yet)
+          style={{ width }}
```
Comment (jpfranci) [inline packages/client/src/pages/common/nav/create-profile-dialog.tsx:116]: I think you just create a `StyledTextField` component and set that as a style. You then just pass in the props to that as well

Comment (danyliu9) [reply packages/client/src/pages/common/nav/create-profile-dialog.tsx]: I ended up making the object InputFieldProps and using spread syntax in the input fields; do you think that fine too or should I try to do it with StyledTextField and StyledFormControl?

**Review by dryu99 — APPROVED** [review]
l g t m

File: packages/client/src/pages/common/nav/create-profile-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            label="Email"
+            value={values.email}
+            onChange={handleChange("email")}
+            required
+            autoFocus
+            variant="standard"
+            {...InputFieldProps}
+          />
+          <TextField
+            id="new-user-username"
```
Comment (jpfranci) [inline packages/client/src/pages/common/nav/create-profile-dialog.tsx:123]: We prob want username to be required

**Pull request merged**